staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/conversion/nop_converter.go

```diff
@@ -61,6 +61,13 @@ func (c *nopConverter) Convert(in, out, context interface{}) error {
 	if err != nil {
 		return err
 	}
+	if context != nil {
+		if groupVersioner, ok := context.(runtime.GroupVersioner); ok {
```

Comment:
do we have a defacto definition of the `context` here? can this be any type we want?

[https://github.com/kubernetes/kubernetes/blob/323e1375b3b15a4144d46d41155ca402a2fcef63/staging/src/k8s.io/apimachinery/pkg/runtime/interfaces.go#L181](https://github.com/kubernetes/kubernetes/blob/323e1375b3b15a4144d46d41155ca402a2fcef63/staging/src/k8s.io/apimachinery/pkg/runtime/interfaces.go#L181)

Reply:
As far as I can tell, there is only one place `context` is not nil. My tooling on finding references to this could be flawed (and I suspect that a lot as it only find 4+3 instances of calling to this interface method).